src/pages/Profile/index.jsx

```diff
@@ -35,64 +38,97 @@ export default class Profile extends Component {
 
   renderBasic() {
     const { userStore, t } = this.props;
-    const { userDetail, changeUser, changeUserRole } = userStore;
+    const { userDetail, changeUser, modifyUser } = userStore;
+    const emailRegexp =
+      "[\\w!#$%&'*+/=?^_`{|}~-]+(?:\\.[\\w!#$%&'*+/=?^_`{|}~-]+)*@(?:[\\w](?:[\\w-]*[\\w])?\\.)[\\w](?:[\\w-]*[\\w])?";
 
     return (
-      <div className={styles.form}>
+      <form className={styles.form} onSubmit={e => modifyUser(e)} method="post">
```

Comment:
no need to pass event to onSubmit

Reply:
This change for prevent page refresh when submit。